indeex.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="stylesheet.css" />
+    <header>
+      <div></div>
+      <div></div>
+      <div class="login">
+        <h3><a href="#">login</a></h3>
+      </div>
+    </header>
+  </head>
+  <body>
+    <!-- container practice -->
+
+    <div class="main-container">
+      <div class="personalinfo">
+        <div class="aboutsec">
+          <h2>About</h2>
+          <p>
+            its Mishal firdous here, i am a software engineer, Previsiosly
+            having one year CMS expertices in wordpress, wix and shopify.Which
+            plays a critical role in the technology industry. Through these
+            content management system design would more eye cautching although
+            responsible for designing, creating, and maintaining software that
+            meets the needs of businesses and users in various domains,
+            including finance, healthcare, e-commerce, entertainment, and many
+            others. Software engineers work with programming languages,
+            algorithms, and software development methodologies to build
+            applications and systems that drive modern society.
+          </p>
+        </div>
+        <div class="experincesec">
+          <h2>Experince</h2>
+        </div>
+      </div>
+
+      <div class="sidebar">
+        <div class="avatar">
+          <img src="assets/avatar.jpeg" />
+          <h2>Mishal Firdous</h2>
+          <ul class="a">
+            <li>
+              <label for="file">Wordpress, Wix, Shopify </label>
+              <progress id="file" value="90" max="100">90%</progress>
+            </li>
+
+            <li>
+              <label for="file">HTML5 ,CSS3, Bootstrap</label>
+              <progress id="file" value="70" max="100">32%</progress>
+            </li>
+
+            <li>
+              <label for="file">Adobe Photo Shop </label>
+              <progress id="file" value="30" max="100">32%</progress>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </body>
+  <footer class="linkedin">
+    <h3>
+      <a href="https://www.linkedin.com/in/mishal-firdous-b99a0a131">Footer</a>
```

Comment:
Please update this to LinkedIn as I suggested previously

Reply:
resolved!